faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/main/MainScreen.kt

```diff
@@ -89,6 +96,7 @@ fun MainScreen(
 
             MainScreenEffect.NavigateToQuran -> navController.navigate(Route.SurRoute)
             MainScreenEffect.NavigateToQiblah -> navController.navigate(Route.CalibrateDeviceRoute)
+            //TODO: Add navigation
             MainScreenEffect.NavigateToMosques -> {}
```

Comment:
could u change it to be 
```suggestion
            MainScreenEffect.NavigateToMosques -> Unit
```